### PR TITLE
fix: update to gta version 2845

### LINKED ISSUE #1031: [Bug]: New GTA Update 2845
### Describe the bug

Rockstar has released a new GTA update 2845. Apparently, the sending of messages has changed a bit.
The pointers for this are oudated.

### Steps To Reproduce

/

### Expected Behavior

/

### Screenshots and/or Logs

```
Failed to find 'RBB'.
Failed to find 'SCM'.
Failed to find 'SCP'.
Failed to find 'HJR'.
```

### Store

Rockstar

### Additional context

_No response_

### Build

- [X] I've confirmed the issue exists on the latest version of YimMenu

### LINKED ISSUE #1033: [Bug]: Failed injection
### Describe the bug

<p align=center>Console and game closes when attempting to inject the menu.</p>

### Steps To Reproduce

<p align=center>1.) Launched GTA 5</p>

<p align=center>2.) Clicked online and launched preferred injector</p>

<p align=center>3.) Loaded YimMenu.dll into the injector and attempted injection</p>

<p align=center>Injection failed with the log below</p>

### Expected Behavior

Menu Injection

### Screenshots and/or Logs

<p align=center><sup><b>

```
[16:42:27.7859128][INFO/main.cpp:61] Yim's Menu Initializing
[16:42:27.7859955][INFO/main.cpp:62] Git Info
	Branch:	master
	Hash:	9056f5aba1c8f045801db7b27c7b4dab44eba30e
	Date:	Thu Feb 23 16:14:44 2023
[16:42:28.0428284][INFO/main.cpp:65] Translation Service initialized.
[16:42:28.0429632][INFO/main.cpp:68] Thread pool initialized.
[16:42:28.0436016][DEBUG/thread_pool.cpp:21] Allocating 8 threads in thread pool.
[16:42:28.0472028][INFO/main.cpp:72] Settings Loaded.
[16:42:28.0483813][INFO/batch.cpp:26] Found 'ISA' GTA5.exe+0xEEC7
[16:42:28.0497067][INFO/batch.cpp:26] Found 'SCREEN_RESOLUTION' GTA5.exe+0x16A2FB
[16:42:28.0498780][INFO/batch.cpp:26] Found 'GS' GTA5.exe+0x2C0B3C
[16:42:28.0544505][INFO/batch.cpp:26] Found 'PF' GTA5.exe+0x72CC08
[16:42:28.0552324][INFO/batch.cpp:26] Found 'FV' GTA5.exe+0x15F4468
[16:42:28.0589872][INFO/batch.cpp:26] Found 'MWL' GTA5.exe+0x5D7CC7
[16:42:28.0646887][INFO/batch.cpp:26] Found 'NPM' GTA5.exe+0x1096780
[16:42:28.0666270][INFO/batch.cpp:26] Found 'SP' GTA5.exe+0xA42375
[16:42:28.0722087][INFO/batch.cpp:26] Found 'ST' GTA5.exe+0x15F743F
[16:42:28.0725683][INFO/batch.cpp:26] Found 'NRF' GTA5.exe+0x1DC9A
[16:42:28.0746000][INFO/batch.cpp:26] Found 'NH' GTA5.exe+0x15F74E9
[16:42:28.0765538][INFO/batch.cpp:26] Found 'S' GTA5.exe+0x1369579
[16:42:28.0771560][INFO/batch.cpp:26] Found 'SG' GTA5.exe+0x15F6D90
[16:42:28.0826189][INFO/batch.cpp:26] Found 'GTS' GTA5.exe+0x15F6F58
[16:42:28.0829161][INFO/batch.cpp:26] Found 'TK' GTA5.exe+0xA464A0
[16:42:28.0849999][INFO/batch.cpp:26] Found 'TSE' GTA5.exe+0xD802BC
[16:42:28.0864221][INFO/batch.cpp:26] Found 'CGSHM' GTA5.exe+0x15FBF09
[16:42:28.0882758][INFO/batch.cpp:26] Found 'RBWD' GTA5.exe+0x668A79
[16:42:28.0897175][INFO/batch.cpp:26] Found 'WMSB' GTA5.exe+0xCED185
[16:42:28.0960269][INFO/batch.cpp:26] Found 'WBD' GTA5.exe+0x14185E8
[16:42:28.1039382][INFO/batch.cpp:26] Found 'WBQ' GTA5.exe+0x130F1A8
[16:42:28.1085615][INFO/batch.cpp:26] Found 'RC' GTA5.exe+0x13E721C
[16:42:28.1091111][INFO/batch.cpp:26] Found 'SEA' GTA5.exe+0x16B1CC9
[16:42:28.1126692][INFO/batch.cpp:26] Found 'WBI64' GTA5.exe+0x1197F91
[16:42:28.1193829][INFO/batch.cpp:26] Found 'RBA' GTA5.exe+0x13073F8
[16:42:28.1194366][INFO/batch.cpp:26] Found 'RI' GTA5.exe+0x1FCC
[16:42:28.1203369][INFO/batch.cpp:26] Found 'WPGSDN' GTA5.exe+0x10EE248
[16:42:28.1255796][INFO/batch.cpp:26] Found 'RBS' GTA5.exe+0x119488C
[16:42:28.1361249][INFO/batch.cpp:26] Found 'WBI32' GTA5.exe+0x1196E63
[16:42:28.1382467][INFO/batch.cpp:26] Found 'PTH' GTA5.exe+0x15FB9C5
[16:42:28.1392603][INFO/batch.cpp:26] Found 'BE' GTA5.exe+0xA6BE3C
[16:42:28.1401306][INFO/batch.cpp:26] Found 'WBB' GTA5.exe+0x1196964
[16:42:28.1436045][INFO/batch.cpp:26] Found 'GSH' GTA5.exe+0x15FCE94
[16:42:28.1472008][INFO/batch.cpp:26] Found 'NPMI' GTA5.exe+0x10F8C7B
[16:42:28.1505187][INFO/batch.cpp:26] Found 'WBA' GTA5.exe+0x16987A2
[16:42:28.1512349][INFO/batch.cpp:26] Found 'REH' GTA5.exe+0x16AE140
[16:42:28.1527314][INFO/batch.cpp:26] Found 'GSCFWC' GTA5.exe+0xA414D0
[16:42:28.1564531][WARN/batch.cpp:32] Failed to find 'RBB'.
[16:42:28.1590850][INFO/batch.cpp:26] Found 'GPR' GTA5.exe+0x11B2AE0
[16:42:28.1650912][INFO/batch.cpp:26] Found 'RNM' GTA5.exe+0x142E5CD
[16:42:28.1692460][INFO/batch.cpp:26] Found 'CNOM' GTA5.exe+0xB18968
[16:42:28.1695416][INFO/batch.cpp:26] Found 'API' GTA5.exe+0x16B4260
[16:42:28.1814333][INFO/batch.cpp:26] Found 'RCC' GTA5.exe+0x1103264
[16:42:28.1816596][INFO/batch.cpp:26] Found 'FR' GTA5.exe+0xCD9014
[16:42:28.1818390][INFO/batch.cpp:26] Found 'MHT' GTA5.exe+0xB202
[16:42:28.1831632][INFO/batch.cpp:26] Found 'GNED' GTA5.exe+0x1A05A20
[16:42:28.1851413][INFO/batch.cpp:26] Found 'RCS/GSTFT/GNOFP/GNO/GSTI' GTA5.exe+0x110421D
[16:42:28.1978738][INFO/batch.cpp:26] Found 'CAD' GTA5.exe+0x1103B02
[16:42:28.1980033][INFO/batch.cpp:26] Found 'WPGDN' GTA5.exe+0x10EED64
[16:42:28.1999910][INFO/batch.cpp:26] Found 'GGCC' GTA5.exe+0xA7616A
[16:42:28.2029946][INFO/batch.cpp:26] Found 'GLT' GTA5.exe+0xDA0A63
[16:42:28.2109965][INFO/batch.cpp:26] Found 'FDGD' GTA5.exe+0x12FACEB
[16:42:28.2115854][INFO/batch.cpp:26] Found 'NPMS' GTA5.exe+0x112D132
[16:42:28.2121376][INFO/batch.cpp:26] Found 'RBIST' GTA5.exe+0x11033E4
[16:42:28.2199714][INFO/batch.cpp:26] Found 'FPFOA' GTA5.exe+0x12FC614
[16:42:28.2209774][INFO/batch.cpp:26] Found 'FPFC' GTA5.exe+0x12F765A
[16:42:28.2226274][INFO/batch.cpp:26] Found 'GVOV' GTA5.exe+0x2E1F2C
[16:42:28.2271988][INFO/batch.cpp:26] Found 'FPFM' GTA5.exe+0x12FDB4E
[16:42:28.2293217][INFO/batch.cpp:26] Found 'FPFUM' GTA5.exe+0x6A208F
[16:42:28.2313147][INFO/batch.cpp:26] Found 'GMI' GTA5.exe+0x164974A
[16:42:28.2319241][INFO/batch.cpp:26] Found 'MCF' GTA5.exe+0x1671F5A
[16:42:28.2397642][INFO/batch.cpp:26] Found 'IMCD' GTA5.exe+0x537975
[16:42:28.2413985][INFO/batch.cpp:26] Found 'SGSBGH' GTA5.exe+0x43CC5D
[16:42:28.2464669][INFO/batch.cpp:26] Found 'N' GTA5.exe+0x10616C1
[16:42:28.2472201][INFO/batch.cpp:26] Found 'SGSBGH' GTA5.exe+0x13F5096
[16:42:28.2513771][INFO/batch.cpp:26] Found 'FDS' GTA5.exe+0x12FD8D8
[16:42:28.2554175][INFO/batch.cpp:26] Found 'JSBI' GTA5.exe+0x11717C4
[16:42:28.2613816][INFO/batch.cpp:26] Found 'VM' GTA5.exe+0x15F6D9A
[16:42:28.2635400][INFO/batch.cpp:26] Found 'RENC' GTA5.exe+0x145A8DE
[16:42:28.2660050][INFO/batch.cpp:26] Found 'SPIDN' GTA5.exe+0x110F7B0
[16:42:28.2671084][INFO/batch.cpp:26] Found 'SDEGSDN' GTA5.exe+0x110D42C
[16:42:28.2695337][INFO/batch.cpp:26] Found 'SVGDN' GTA5.exe+0x1116F38
[16:42:28.2811514][INFO/batch.cpp:26] Found 'GVGAS' GTA5.exe+0x10F11E4
[16:42:28.2860212][WARN/batch.cpp:32] Failed to find 'SCM'.
[16:42:28.2877851][INFO/batch.cpp:26] Found 'C' GTA5.exe+0xCDB040
[16:42:28.2916769][INFO/batch.cpp:26] Found 'GU' GTA5.exe+0x1413C06
[16:42:28.2952241][INFO/batch.cpp:26] Found 'HT' GTA5.exe+0x13E7E69
[16:42:28.3002502][INFO/batch.cpp:26] Found 'PGI' GTA5.exe+0x14044B8
[16:42:28.3027034][INFO/batch.cpp:26] Found 'APTS' GTA5.exe+0x11E6B94
[16:42:28.3054234][INFO/batch.cpp:26] Found 'HJR' GTA5.exe+0x117654C
[16:42:28.3086969][INFO/batch.cpp:26] Found 'PMFR' GTA5.exe+0x13FF66C
[16:42:28.3090018][INFO/batch.cpp:26] Found 'QP' GTA5.exe+0x145297A
[16:42:28.3091920][INFO/batch.cpp:26] Found 'SGS' GTA5.exe+0x118A508
[16:42:28.3100458][INFO/batch.cpp:26] Found 'SPDM' GTA5.exe+0x111939C
[16:42:28.3102953][INFO/batch.cpp:26] Found 'SCNM' GTA5.exe+0x11E6F54
[16:42:28.3149636][INFO/batch.cpp:26] Found 'PIGI' GTA5.exe+0x10A249F
[16:42:28.3220276][INFO/batch.cpp:26] Found 'SJRM' GTA5.exe+0x1273EC6
[16:42:28.3250312][INFO/batch.cpp:26] Found 'IMSV' GTA5.exe+0x116FAC4
[16:42:28.3311657][INFO/batch.cpp:26] Found 'GCP&SRGC' GTA5.exe+0x11E7DB1
[16:42:28.3319589][INFO/batch.cpp:26] Found 'SND' GTA5.exe+0xF4C9F3
[16:42:28.3388697][INFO/batch.cpp:26] Found 'RC' GTA5.exe+0xF7CD40
[16:42:28.3401719][INFO/batch.cpp:26] Found 'BP' GTA5.exe+0x16BE3EE
[16:42:28.3407473][INFO/batch.cpp:26] Found 'HRGC' GTA5.exe+0x11E0AE6
[16:42:28.3412664][INFO/batch.cpp:26] Found 'SSMA' GTA5.exe+0x13F79C0
[16:42:28.3414536][INFO/batch.cpp:26] Found 'STOPVT' GTA5.exe+0x699468
[16:42:28.3419309][INFO/batch.cpp:26] Found 'CW' GTA5.exe+0x16487F
[16:42:28.3423142][INFO/batch.cpp:26] Found 'CD' GTA5.exe+0x18CFDF
[16:42:28.3435564][INFO/batch.cpp:26] Found 'CSH' GTA5.exe+0xA337E0
[16:42:28.3453745][INFO/batch.cpp:26] Found 'IDC' GTA5.exe+0x67594C
[16:42:28.3502558][INFO/batch.cpp:26] Found 'CAC' GTA5.exe+0x1480C5C
[16:42:28.3536676][WARN/batch.cpp:32] Failed to find 'HJR'.
[16:42:28.3627507][INFO/batch.cpp:26] Found 'DPI' GTA5.exe+0x1433860
[16:42:28.3686909][INFO/batch.cpp:26] Found 'BNA' GTA5.exe+0x16BE204
[16:42:28.3692387][INFO/batch.cpp:26] Found 'RR' GTA5.exe+0xC0B6A7
[16:42:28.3744002][INFO/batch.cpp:26] Found 'LCF' GTA5.exe+0x5D5EDC
[16:42:28.3783453][INFO/batch.cpp:26] Found 'MFO' GTA5.exe+0xA6EF53
[16:42:28.3810656][INFO/batch.cpp:26] Found 'SAACF' GTA5.exe+0xA8A11C
[16:42:28.3829318][INFO/batch.cpp:26] Found 'ESI' GTA5.exe+0x10EEE86
[16:42:28.3877643][INFO/batch.cpp:26] Found 'SJD' GTA5.exe+0xA89CC0
[16:42:28.3913526][INFO/batch.cpp:26] Found 'SCI' GTA5.exe+0x13D1EA7
[16:42:28.3927288][INFO/batch.cpp:26] Found 'TPO270' GTA5.exe+0xBB5CE6
[16:42:28.3966183][INFO/batch.cpp:26] Found 'DSI' GTA5.exe+0x13F6B44
[16:42:28.4032792][INFO/batch.cpp:26] Found 'NT' GTA5.exe+0xCCB286
[16:42:28.4038878][INFO/batch.cpp:26] Found 'PMFS' GTA5.exe+0x13E301C
[16:42:28.4082513][INFO/batch.cpp:26] Found 'SP' GTA5.exe+0x1450338
[16:42:28.4084805][INFO/batch.cpp:26] Found 'CTP' GTA5.exe+0x14426D0
[16:42:28.4118749][INFO/batch.cpp:26] Found 'ICF' GTA5.exe+0xA9ED6B
[16:42:28.4156000][INFO/batch.cpp:26] Found 'RS' GTA5.exe+0x35F8773
[16:42:28.4182141][INFO/batch.cpp:26] Found 'ITC' GTA5.exe+0x29D13B
[16:42:28.4199496][INFO/batch.cpp:26] Found 'SOD' GTA5.exe+0x128B247
[16:42:28.4252841][INFO/batch.cpp:26] Found 'GEAT' GTA5.exe+0xA88B7C
[16:42:28.4339307][INFO/batch.cpp:26] Found 'RPI' GTA5.exe+0x6B04A0
[16:42:28.4357993][INFO/batch.cpp:26] Found 'NTQVMC' GTA5.exe+0x151787D
[16:42:28.4361794][INFO/batch.cpp:26] Found 'FPC2' GTA5.exe+0x147D0D7
[16:42:28.4362416][INFO/batch.cpp:26] Found 'SNT' GTA5.exe+0x145B0C1
[16:42:28.4364055][INFO/batch.cpp:26] Found 'RAU' GTA5.exe+0x16AF3E0
[16:42:28.4382393][INFO/batch.cpp:26] Found 'WPCDN' GTA5.exe+0x10E0BD8
[16:42:28.4396486][INFO/batch.cpp:26] Found 'QD' GTA5.exe+0x12E526C
[16:42:28.4404201][WARN/batch.cpp:32] Failed to find 'SCP'.
[16:42:28.4452152][INFO/batch.cpp:26] Found 'CPTN' GTA5.exe+0xD52E24
[16:42:28.4499913][INFO/batch.cpp:26] Found 'FPC' GTA5.exe+0x147CEA8
[16:42:28.4518796][ERROR/exception_handler.cpp:27] UNKNOWN_EXCEPTION
Dumping modules:
GTA5.exe Base Address: 0x7FF6D48E0000 Size: 63526912
ntdll.dll Base Address: 0x7FF8E51F0000 Size: 1154560
KERNEL32.DLL Base Address: 0x7FF8E4090000 Size: 519168
KERNELBASE.dll Base Address: 0x7FF8E2D40000 Size: 1134080
USER32.dll Base Address: 0x7FF8E3230000 Size: 593920
win32u.dll Base Address: 0x7FF8E2AD0000 Size: 42496
GDI32.dll Base Address: 0x7FF8E36B0000 Size: 58880
gdi32full.dll Base Address: 0x7FF8E28E0000 Size: 648704
msvcp_win.dll Base Address: 0x7FF8E2B00000 Size: 344064
ucrtbase.dll Base Address: 0x7FF8E2BA0000 Size: 735744
PSAPI.DLL Base Address: 0x7FF8E3490000 Size: 2048
WS2_32.dll Base Address: 0x7FF8E3D50000 Size: 275456
RPCRT4.dll Base Address: 0x7FF8E34A0000 Size: 919040
CRYPT32.dll Base Address: 0x7FF8E3020000 Size: 1071104
WINTRUST.dll Base Address: 0x7FF8E2CA0000 Size: 299520
msvcrt.dll Base Address: 0x7FF8E4150000 Size: 477184
IMM32.dll Base Address: 0x7FF8E3890000 Size: 126464
VERSION.dll Base Address: 0x7FF8DB050000 Size: 10752
bcrypt.dll Base Address: 0x7FF8E2D10000 Size: 103424
SHLWAPI.dll Base Address: 0x7FF8E41F0000 Size: 182784
ADVAPI32.dll Base Address: 0x7FF8E37E0000 Size: 423424
sechost.dll Base Address: 0x7FF8E36E0000 Size: 412672
SHELL32.dll Base Address: 0x7FF8E46E0000 Size: 5804544
ole32.dll Base Address: 0x7FF8E4490000 Size: 832000
combase.dll Base Address: 0x7FF8E4E50000 Size: 2331136
OLEAUT32.dll Base Address: 0x7FF8E3FC0000 Size: 608256
DSOUND.dll Base Address: 0x7FF8C29D0000 Size: 380416
WININET.dll Base Address: 0x7FF8B3850000 Size: 2026496
MF.dll Base Address: 0x7FF8C1450000 Size: 340992
WINMM.dll Base Address: 0x7FF8D63D0000 Size: 81408
MFPlat.DLL Base Address: 0x7FF8B26E0000 Size: 1409536
cfgmgr32.dll Base Address: 0x7FF8E2A80000 Size: 218624
msdmo.dll Base Address: 0x7FF8BBE10000 Size: 13312
shcore.dll Base Address: 0x7FF8E3EB0000 Size: 476672
PROPSYS.dll Base Address: 0x7FF8DE450000 Size: 631296
MFReadWrite.dll Base Address: 0x7FF87ECC0000 Size: 918016
WTSAPI32.dll Base Address: 0x7FF8DE430000 Size: 37888
d3d9.dll Base Address: 0x7FF8DD540000 Size: 1373184
D3DCOMPILER_43.dll Base Address: 0x7FF8B4B80000 Size: 2423296
DINPUT8.dll Base Address: 0x7FF8D72E0000 Size: 169472
XINPUT1_3.dll Base Address: 0x400000 Size: 85504
SETUPAPI.dll Base Address: 0x7FF8E38E0000 Size: 864768
IPHLPAPI.DLL Base Address: 0x7FF8E1C80000 Size: 166400
powrprof.dll Base Address: 0x7FF8E2750000 Size: 70656
winmmbase.dll Base Address: 0x7FF8A80D0000 Size: 95232
kernel.appcore.dll Base Address: 0x7FF8E0780000 Size: 16384
dwmapi.dll Base Address: 0x7FF8E0490000 Size: 99328
windows.storage.dll Base Address: 0x7FF8E0980000 Size: 5820416
Wldp.dll Base Address: 0x7FF8E2260000 Size: 105472
bink2w64.dll Base Address: 0x7FF8C1DF0000 Size: 307712
GFSDK_TXAA_AlphaResolve.win64.dll Base Address: 0x7FF8D73D0000 Size: 48128
GFSDK_ShadowLib.win64.dll Base Address: 0x7FF8977A0000 Size: 96256
OPENGL32.dll Base Address: 0x7FF8BE230000 Size: 909312
MFCORE.DLL Base Address: 0x7FF87D700000 Size: 3960832
GLU32.dll Base Address: 0x7FF8D75B0000 Size: 122880
ksuser.dll Base Address: 0x7FF8C5810000 Size: 6144
d3dx9_43.dll Base Address: 0x7FF8B0CF0000 Size: 2280960
CRYPTBASE.DLL Base Address: 0x7FF8E21B0000 Size: 10752
UMPDC.dll Base Address: 0x7FF8E2730000 Size: 34816
bcryptPrimitives.dll Base Address: 0x7FF8E29F0000 Size: 408576
RTWorkQ.DLL Base Address: 0x7FF8D5D30000 Size: 105472
MSASN1.dll Base Address: 0x7FF8E23F0000 Size: 34816
gameoverlayrenderer64.dll Base Address: 0x7FF8C0CA0000 Size: 982528
inputhost.dll Base Address: 0x7FF8D48F0000 Size: 976384
CoreUIComponents.dll Base Address: 0x7FF8DFA10000 Size: 1804800
ntmarta.dll Base Address: 0x7FF8E19E0000 Size: 140288
wintypes.dll Base Address: 0x7FF8DF7B0000 Size: 487424
CoreMessaging.dll Base Address: 0x7FF8DFDA0000 Size: 606720
sdk_legacy_led_x64.dll Base Address: 0x7FF896550000 Size: 2538496
VCRUNTIME140.dll Base Address: 0x7FF8DAD80000 Size: 65024
VCRUNTIME140_1.dll Base Address: 0x7FF8DBC00000 Size: 15360
MSVCP140.dll Base Address: 0x7FF8C18D0000 Size: 346112
CRYPTSP.dll Base Address: 0x7FF8E21C0000 Size: 46080
rsaenh.dll Base Address: 0x7FF8E18A0000 Size: 142336
mswsock.dll Base Address: 0x7FF8E1FA0000 Size: 331776
dbghelp.dll Base Address: 0x7FF8D8D70000 Size: 1400320
dbgcore.DLL Base Address: 0x7FF8D8D10000 Size: 142336
uxtheme.dll Base Address: 0x7FF8E02B0000 Size: 382464
clbcatq.dll Base Address: 0x7FF8E33E0000 Size: 437760
amsi.dll Base Address: 0x7FF8D67F0000 Size: 62464
USERENV.dll Base Address: 0x7FF8E27E0000 Size: 108032
profapi.dll Base Address: 0x7FF8E2820000 Size: 68608
MpOav.dll Base Address: 0x7FF8D6220000 Size: 204800
dxcore.dll Base Address: 0x7FF8D8C40000 Size: 147968
cryptnet.dll Base Address: 0x7FF8DB010000 Size: 135168
drvstore.dll Base Address: 0x7FF8DA4C0000 Size: 979456
devobj.dll Base Address: 0x7FF8E25B0000 Size: 109056
imagehlp.dll Base Address: 0x7FF8E4E30000 Size: 45056
nvspcap64.dll Base Address: 0x7FF8B4750000 Size: 2055680
gpapi.dll Base Address: 0x7FF8E1130000 Size: 69120
WINNSI.DLL Base Address: 0x7FF8DB920000 Size: 11776
NSI.dll Base Address: 0x7FF8E3EA0000 Size: 7680
steam_api64.dll Base Address: 0x6D260000 Size: 110592
steamclient64.dll Base Address: 0x7FF877290000 Size: 15837696
vstdlib_s64.dll Base Address: 0x7FF8C4B80000 Size: 285184
Secur32.dll Base Address: 0x7FF8CA710000 Size: 11264
tier0_s64.dll Base Address: 0x7FF8B91C0000 Size: 257536
SSPICLI.DLL Base Address: 0x7FF8E27A0000 Size: 123392
D3D11.DLL Base Address: 0x7FF8DEB70000 Size: 1853952
dxgi.dll Base Address: 0x7FF8E1160000 Size: 614912
nvapi64.dll Base Address: 0x7FF8B6820000 Size: 6148608
D3D10_1.DLL Base Address: 0x7FF8CBA00000 Size: 130048
d3d10_1core.dll Base Address: 0x7FF8DAF60000 Size: 7680
MSCTF.dll Base Address: 0x7FF8E45C0000 Size: 864768
textinputframework.dll Base Address: 0x7FF8D4A50000 Size: 724992
Oleacc.dll Base Address: 0x7FF8DCBE0000 Size: 260608
WINSTA.dll Base Address: 0x7FF8E2610000 Size: 219648
nvldumdx.dll Base Address: 0x7FF8C5D90000 Size: 599552
nvwgf2umx.dll Base Address: 0x7FF8722F0000 Size: 33575936
NvCamera64.dll Base Address: 0x7FF87EE30000 Size: 4086784
WindowsCodecs.dll Base Address: 0x7FF8DBDE0000 Size: 1328128
HID.DLL Base Address: 0x7FF8E1120000 Size: 20480
XINPUT9_1_0.dll Base Address: 0x7FF8DCBB0000 Size: 4096
d3dcompiler_47_64.dll Base Address: 0x7FF8947E0000 Size: 3922944
MMDevApi.dll Base Address: 0x7FF8DBD50000 Size: 387072
XAudio2_7.dll Base Address: 0x7FF8C0ED0000 Size: 485376
AUDIOSES.DLL Base Address: 0x7FF8DC010000 Size: 1037312
resourcepolicyclient.dll Base Address: 0x7FF8E0390000 Size: 31744
Windows.UI.dll Base Address: 0x7FF8D4B50000 Size: 731136
WindowManagementAPI.dll Base Address: 0x7FF8DC750000 Size: 450048
twinapi.appcore.dll Base Address: 0x7FF8DC2B0000 Size: 1449984
avrt.dll Base Address: 0x7FF8DCFC0000 Size: 5632
MessageBus.dll Base Address: 0x7FF8C9FA0000 Size: 5349888
dhcpcsvc.DLL Base Address: 0x7FF8DB150000 Size: 61440
napinsp.dll Base Address: 0x7FF8B02E0000 Size: 50176
pnrpnsp.dll Base Address: 0x7FF8B02C0000 Size: 66048
wshbth.dll Base Address: 0x7FF8DDA20000 Size: 43008
NLAapi.dll Base Address: 0x7FF8DB990000 Size: 68608
DNSAPI.dll Base Address: 0x7FF8E1CC0000 Size: 599040
winrnr.dll Base Address: 0x7FF8B02A0000 Size: 32768
fwpuclnt.dll Base Address: 0x7FF8D6E30000 Size: 286720
rasadhlp.dll Base Address: 0x7FF8D5980000 Size: 7168
socialclub.dll Base Address: 0x7FF87FDD0000 Size: 5077504
chrome_elf.dll Base Address: 0x7FF8BEEE0000 Size: 685568
libcef.dll Base Address: 0x7FF84DE50000 Size: 109755392
COMDLG32.dll Base Address: 0x7FF8E3DC0000 Size: 684032
NETAPI32.dll Base Address: 0x7FF8CE310000 Size: 47616
UIAutomationCore.DLL Base Address: 0x7FF8A8A20000 Size: 2040832
COMCTL32.dll Base Address: 0x7FF8C54A0000 Size: 560128
WINSPOOL.DRV Base Address: 0x7FF8C79D0000 Size: 351744
USP10.dll Base Address: 0x7FF8A4EA0000 Size: 512
DWrite.dll Base Address: 0x7FF8CDC90000 Size: 1601024
dxva2.dll Base Address: 0x7FF8A50C0000 Size: 96256
credui.dll Base Address: 0x7FF8A50A0000 Size: 62464
WINHTTP.dll Base Address: 0x7FF8D9890000 Size: 850944
urlmon.dll Base Address: 0x7FF8CE510000 Size: 1279488
CRYPTUI.dll Base Address: 0x7FF8C7AD0000 Size: 246272
ncrypt.dll Base Address: 0x7FF8E22D0000 Size: 98304
wevtapi.dll Base Address: 0x7FF8DCF30000 Size: 296960
ESENT.dll Base Address: 0x7FF8C6B90000 Size: 2553344
iertutil.dll Base Address: 0x7FF8D3210000 Size: 550400
srvcli.dll Base Address: 0x7FF8CE870000 Size: 54272
netutils.dll Base Address: 0x7FF8E1D90000 Size: 19456
WKSCLI.DLL Base Address: 0x7FF8E19C0000 Size: 45568
SAMCLI.DLL Base Address: 0x7FF8D8C90000 Size: 61440
NTASN1.dll Base Address: 0x7FF8E2290000 Size: 81408
DiscordHook64.dll Base Address: 0x7FF8BAF00000 Size: 762880
D3DCompiler_47.dll Base Address: 0x7FF8DEDE0000 Size: 3396608
FirewallAPI.dll Base Address: 0x7FF8E14C0000 Size: 364544
fwbase.dll Base Address: 0x7FF8E1450000 Size: 157184
FWPolicyIOMgr.dll Base Address: 0x7FF8CE4B0000 Size: 216064
wdmaud.drv Base Address: 0x7FF8BBAA0000 Size: 193024
msacm32.drv Base Address: 0x7FF8BCC90000 Size: 17920
MSACM32.dll Base Address: 0x7FF8BBA80000 Size: 74240
midimap.dll Base Address: 0x7FF8BBA70000 Size: 13312
ondemandconnroutehelper.dll Base Address: 0x7FF8B0310000 Size: 42496
dhcpcsvc6.DLL Base Address: 0x7FF8DA7E0000 Size: 23552
wbemprox.dll Base Address: 0x7FF8D8CD0000 Size: 27648
wbemcomn.dll Base Address: 0x7FF8D2AB0000 Size: 328704
wbemsvc.dll Base Address: 0x7FF8D63B0000 Size: 7168
fastprox.dll Base Address: 0x7FF8D6400000 Size: 708608
YimMenu.dll Base Address: 0x7FF8957A0000 Size: 2207232
schannel.DLL Base Address: 0x7FF8E17C0000 Size: 487936
mskeyprotect.dll Base Address: 0x7FF8BE150000 Size: 44544
ncryptsslp.dll Base Address: 0x7FF8BE680000 Size: 78336
Dumping registers:
RAX: 0xFF6
RCX: 0x22235EE0000
RDX: 0x22205010004
RBX: 0x7FF895AEB6C8
RSI: 0xE098C1DB70
RDI: 0x19930520
RSP: 0xE098C1DA10
RBP: 0xE098C1DC50
R8:  0x7FF8E51FE3A9
R9:  0x2216CE40000
R10: 0xE098C1D370
R11: 0x0
R12: 0x0
R13: 0x2223699C2F0
R14: 0x0
R15: 0x0
Dumping stacktrace:
[0]	KERNELBASE.dll RaiseException
[1]	VCRUNTIME140.dll CxxThrowException
[2]	YimMenu.dll+0xCDD84 0x7FF89586DD84
[3]	YimMenu.dll+0xBB200 0x7FF89585B200
[4]	YimMenu.dll+0xBBEA8 0x7FF89585BEA8
[5]	YimMenu.dll+0xBA811 0x7FF89585A811
[6]	KERNEL32.DLL BaseThreadInitThunk
[7]	ntdll.dll RtlUserThreadStart
--------End of exception--------
```
</b></sup></p>

### Store

Steam

### Additional context

_No response_

### Build

- [X] I've confirmed the issue exists on the latest version of YimMenu

### PR DESCRIPTION
**Pull request by Primexz**
YimMenu is now working again.
If my pointers are not correct, please tell me!

~~I had not managed to update the m_send_chat_ptr sig. This still needs to be done.~~

fixes #1033 
fixes #1031

### CODE REVIEW

File: src/pointers.cpp [hunk 1]
```diff
@@ -167,9 +167,9 @@ namespace big
 		});
 
 		// Read Bitbuffer Boolean
-		main_batch.add("RBB", "E8 ? ? ? ? 84 C0 74 2D 48 8D 57 20", [this](memory::handle ptr)
+		main_batch.add("RBB", "48 8B C4 48 89 58 08 55 56 57 48 83 EC 20 48 83 6", [this](memory::handle ptr)
```
Comment (MarkEcza) [inline src/pointers.cpp]: E8 ? ? ? ? 84 C0 74 41 48 8D 56 2C +1 rip

Comment (Primexz) [reply src/pointers.cpp]: thx!

**Review by Yimura — DISMISSED** [review]

File: src/pointers.cpp [hunk 2]
```diff
@@ -418,6 +418,12 @@ namespace big
 			m_invalid_mods_crash_detour = ptr.add(1).rip().as<PVOID>();
 		});
 
+		// Send Chat Ptr
+		main_batch.add("SCP", "83 7E 1C 01 48 8B 3D", [this](memory::handle ptr)
```
Comment (Seanghost117) [inline src/pointers.cpp:422]: think this is a bad sig

Comment (Primexz) [reply src/pointers.cpp:422]: yep, could not update the whole thing correctly yet, as described above

Comment (Seanghost117) [reply src/pointers.cpp:422]: disregard

Comment (Yimura) [reply src/pointers.cpp:422]: It's add(7).rip() I believe, haven't checked out the code or tested it.

Old:
image
New:
image

Comment (Primexz) [reply src/pointers.cpp:422]: pretty good, works!

**Comment by Primexz** [comment]
pr is ready.

**Comment by ouka1998** [comment]
nice sir

**Review by Yimura — APPROVED** [review]

**Pull request merged**